corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -733,15 +734,20 @@ def __repr__(self):
 
     def render(self, context):
         if self.name not in context and self.value:
+            # Check that there isn't already an entry point from the other bundler tool
+            # If there is, don't add this one, because having both set will cause js errors
+            other_tag = "js_entry" if self.name == "requirejs_main" else "requirejs_main"
+            other_value = None
+            for context_dict in context.dicts:
+                if other_tag in context_dict:
+                    other_value = context_dict.get(other_tag)
```

Comment:
nit: bracket syntax can be used here since `other_tag in context_dict` was true
```suggestion
                    other_value = context_dict[other_tag]
```
`dict.get(key)` returns `None` if the key is missing, which would cause `if not other_value:` to have a surprising outcome below. Bracket syntax avoids that ambiguity.